sdcm/fill_db_data.py

```diff
@@ -436,7 +436,7 @@ class FillDatabaseData(ClusterTester):
                 [[-4]]
             ],
             'min_version': '1.7',
-            'skip_condition': "self.params.get('experimental')",
+            'skip_condition': "'experimental_features' in self.params",
```

Comment:
I believe we can remove this logic althoughter

Reply:
dropped.